app/views/layouts/_header.html.haml

```diff
@@ -0,0 +1,16 @@
+%header
+  .menu-wrapper
+    %a.logo{:href => root_path}
+      %span.logo-icon
+        Insert logo here
+
+    = link_to "Postear nuevo evento", new_event_path
```

Comment:
Why do you have this text in spanish while every other text is in english? Here we should keep every text/label in english or spanish. or maybe localize the application.